examples/04-Retrieval-Model.ipynb

```diff
@@ -0,0 +1,1847 @@
+{
```

Comment:
<p>good question. I think since originally a dask-cudf df is returned from <code>unique_rows_by_features</code>,  in the case of OOM, we might not convert it to <code>cudf</code> with compute() but we can save it on disk where each partitions will be saved as a parquet file. </p>